.github/scripts/analyze-builds.js

```diff
@@ -0,0 +1,111 @@
+const { resolve } = require("path");
+const { promisify } = require("util");
+const {
+  createWriteStream,
+  promises: { readdir, readFile }
+} = require("fs");
+const exec = promisify(require("child_process").exec);
+
+const EXAMPLES_PATH = "../../esm-samples";
+
+const buildInfo = {
+  "jsapi-angular-cli": {
+    buildDirectory: "dist",
+    bundleDirectory: "./",
+    title: "Angular 13"

```

Comment:
I'm not sure about having the version number in `title` because they iterate a lot.  Is there a property we can read from the package.json file instead?

Reply:
Is this what you had in mind? https://github.com/benelan/test/commit/5d18ac8641d1a5693e29bc677246621d1be6d851